src/main/java/com/company/zayats/hw11/Main.java

```diff
@@ -0,0 +1,6 @@
+package com.company.zayats.hw11;
+
+public class Main {
+    public static void main(String[] args) {
+    }
```

Comment:
Please add some code for how you use your methods, sometimes this is included in the assessment